moped-editor/src/views/projects/projectView/NoteInput.js

```diff
@@ -219,8 +239,8 @@ const NoteInput = ({
               </div>
             )}
             <ProjectSaveButton
-              // disable save button if no text after removing html tags
-              disabled={!noteText.replace(/<[^>]*>/g, "")}
```

Comment:
nice that you were able to remove this regex! I confirmed that I was unable to save a note that looks empty in the editor but writes empty HTML tags into the database.

Reply:
yay thanks for confirming that! :)